src/utils_project.js

```diff
@@ -400,7 +400,7 @@ go.utils_project = {
         };
 
         return go.utils
-            .service_api_call("rating", "post", null, payload, "rating/", im)
+            .service_api_call("service_rating", "post", null, payload, "service_rating/", im)
```

Comment:
just the name of the service in the config needed to change, the second variable still needs to point to the `rating` app.